src/kili/services/export/format/coco/__init__.py

```diff
@@ -429,7 +432,7 @@ def _get_coco_image_annotations(
             # But a type of object can appear several times on the same image.
             # The limitation of the single connected part comes from Kili.
             segmentation=[poly],
-            area=coco_image["height"] * coco_image["width"],
+            area=bbox[2] * bbox[3],
```

Comment:
I don't think that this area is ok for an annotation that is not a bounding box. It is supposed to be the area of the annotation and not the area of the bounding box surrounding the shape.

Reply:
Hum, you're right, do we you know how to compute easily the area of complex forms?